src/onboarding/onboarding-integrations.ts

```diff
@@ -161,10 +161,12 @@ class OnboardingIntegrations extends LitElement {
 
   private async _loadConfigEntries() {
     const entries = await getConfigEntries(this.hass!);
-    // We filter out the config entry for the local weather.
+    // We filter out the config entry for the local weather and rpi_power.
     // It is one that we create automatically and it will confuse the user
     // if it starts showing up during onboarding.
-    this._entries = entries.filter((entry) => entry.domain !== "met");
+    this._entries = entries.filter(
+      (entry) => entry.domain !== "met" && entry.domain !== "rpi_power"
+    );
```

Comment:
Put `HIDDEN_DOMAINS` in the top of the file, outside the class.
```suggestion
    const HIDDEN_DOMAINS = new Set(["met", "rpi_power"]);
    this._entries = entries.filter(
      (entry) => !HIDDEN_DOMAINS.has(entry.domain)
    );
```